### DISCUSSION TITLE
NextAuth Supabase adapter error with "Content-Type not acceptable" message

### DISCUSSION THREAD

**Discussion by dev-wiillus** [supabase/supabase#12650] [Questions] [answered]
Hi, I'm currently working on a Next.js project and using the NextAuth Supabase adapter for authentication. However, I'm encountering an issue where the adapter is returning an error with the message "Content-Type not acceptable: application/json, text/plain". The error code is "PGRST102" which seems to indicate a Postgres-related issue.

I follwed [NextAuth document](https://next-auth.js.org/adapters/supabase) exactly, google login had issue with Google sign-in. 

Has anyone encountered a similar issue with the NextAuth Supabase adapter? If so, how did you resolve it? Any help or suggestions would be greatly appreciated. Thanks in advance!

Error message:
```

[next-auth][error][adapter_error_createUser] 
https://next-auth.js.org/errors#adapter_error_createuser Content-Type not acceptable: application/json, text/plain {
  code: 'PGRST102',
  details: null,
  hint: null,
  message: 'Content-Type not acceptable: application/json, text/plain
}

```

**Comment by steve-chavez** [comment, marked as answer]
This is a bug in Node.js, upgrading would solve the issue. See https://github.com/supabase/supabase-js/issues/691.

**Reply by robin-wiillus** [reply]
Thank you so much steve!
My problem solved:)
